js/main.js

```diff
@@ -5,17 +5,26 @@ var $location = document.querySelector('#location');
 var bio = document.querySelector('#bio');
 var image = document.querySelector('.image');
 var form = document.querySelector('form');
+
 var $views = {
   editProfile: document.querySelector('[data-view="edit-profile"]'),
-  viewProfile: document.querySelector('[data-view="view-profile"')
+  viewProfile: document.querySelector('[data-view="view-profile"]'),
+  profileButton: document.querySelector('[data-view="profile"]')
 };
 
+var h1;
+var row;
+
 avatar.addEventListener('input', function () {
   image.setAttribute('src', avatar.value);
 });
 
 form.addEventListener('submit', function (event) {
   event.preventDefault();
+  if (data.profile.username !== '') {
+    $views.viewProfile.removeChild(h1);
+    $views.viewProfile.removeChild(row);
+  }
```

Comment:
instead of removing elements, just hide them by default, since its in the navbar you can store a reference to it, and just hard code it in the html